SQL/02_Tabloid_Seed_Data.sql

```diff
@@ -17,8 +17,8 @@ values (1, 'C#'), (2, 'JavaScript'), (3, 'Funny'), (4, 'Scary');
 set identity_insert [Tag] off
```

Comment:
This file should not be included in pull requests

Reply:
Deleted file from my branch!